### PR TITLE
LPS-130454 Error exporting uses when working with user custom fields

### PR DESCRIPTION
**Pull request by pei-jung**

### CODE REVIEW

**Comment by davidtellolr** [comment]
Hi @pei-jung ,

Did you have the opportunity to review this PR?

Please, let me know if you have any doubt.

Regards.

**Comment by davidtellolr** [comment]
Hi @pei-jung ,

Sorry for my previous message. I didn't see your conversation with Mariano. Please forget it.

Thanks and regards!

**Comment by Sharryxue** [comment]
Hello @pei-jung,
The new added integration test is failed with following error message, would you please take a look into it? You can refer the [jenkins console](https://testray.liferay.com/reports/production/logs/2021-05/test-1-22/test-portal-acceptance-pullrequest(master)/8178/modules-integration-mysql57-jdk8/0/1/jenkins-console.txt.gz), [testray report](https://testray.liferay.com/home/-/testray/case_results/1533984603) for more details, thanks in advance.
```
     [exec] com.liferay.users.admin.internal.portlet.action.test.ExportUsersMVCResourceCommandTest > testGetUserCSVWithExpando STARTED
     [exec] 2021-05-21 09:33:41.105 INFO  [com.liferay.users.admin.test-executor-thread][CompanyIndexFactory:57] liferay-43876
     [exec] 2021-05-21 09:33:41,120 com.liferay.users.admin.test-executor-thread ERROR Attempted to append to non-started appender pkHgwMaE
     [exec] 
     [exec] com.liferay.users.admin.internal.portlet.action.test.ExportUsersMVCResourceCommandTest > testGetUserCSVWithExpando FAILED
     [exec]     java.lang.RuntimeException: java.lang.IllegalArgumentException: Someone may be trying to circumvent the permission checker: {companyId=20103, name=com.liferay.expando.kernel.model.ExpandoColumn, primKey=46127, scope=4}
     [exec]     	at com.liferay.portlet.expando.model.impl.ExpandoBridgeImpl.getAttribute(ExpandoBridgeImpl.java:233)
     [exec]     	at com.liferay.portlet.expando.model.impl.ExpandoBridgeImpl.getAttribute(ExpandoBridgeImpl.java:213)
     [exec]     	at com.liferay.users.admin.web.internal.portlet.action.ExportUsersMVCResourceCommand.getUserCSV(ExportUsersMVCResourceCommand.java:116)
```

**Comment by pei-jung** [comment]
ci:test:sf

**Comment by pei-jung** [comment]
ci:test:relevant

**Review by drewbrokke — COMMENTED** [review]
@pei-jung the test case seems good. I still think it would be best to move this portal property to a company-scoped configuration.

If the issue is urgent, this change seems fine to merge, but let's make sure to create a `technical-debt` ticket for migrating the property to a config. It also is in line with the other efforts of LOL to make more features support multi-tenancy.

File: modules/apps/users-admin/users-admin-test/src/testIntegration/java/com/liferay/users/admin/internal/portlet/action/test/ExportUsersMVCResourceCommandTest.java [hunk 1]
```diff
@@ -0,0 +1,141 @@
...
+				new String[] {
+					"fullName", "expando:" + expandoColumn.getName()
+				});
+
+			StringBundler sb1 = new StringBundler(4);
+
+			sb1.append(CSVUtil.encode(user1.getFullName()));
+			sb1.append(StringPool.COMMA);
+			sb1.append(CSVUtil.encode(expandoValue.getString()));
+			sb1.append(StringPool.NEW_LINE);
```
[the remark this answers is not in the document]
Comment (pei-jung) [reply modules/apps/users-admin/users-admin-test/src/testIntegration/java/com/liferay/users/admin/internal/portlet/action/test/ExportUsersMVCResourceCommandTest.java:97]: Makes sense. I will update the code to use `StringBundler.concat`

**Comment by pei-jung** [comment]
@drewbrokke fyi, I created a ticket for migrating the property to a company scoped config: https://issues.liferay.com/browse/LPS-132603

**Comment by pei-jung** [comment]
ci:forward

**Comment by Sharryxue** [comment]
Failure not related to the commits made here, retrigger the forward test.

**Comment by Sharryxue** [comment]
ci:forward

**Comment by Sharryxue** [comment]
The new failure not related, it also reproducible on other prs, manually forwarded the pr [brianchandotcom/liferay-portal#102162](https://github.com/brianchandotcom/liferay-portal/pull/102162).

**Comment by Sharryxue** [comment]
ci:close

**Pull request closed without merging**